src/RequestForm.js

```diff
@@ -517,7 +533,7 @@ class RequestForm extends React.Component {
       pickupServicePointId,
     } = data;
 
-    const [block] = this.getPatronBlocks(parentResources);
+    const [block = {}] = this.getPatronBlocks(parentResources);
```

Comment:
I think it's better to remove the default value here and then make the condition look like:
```
if (block && block.userId === this.state.selectedUser.id) 
```
or
```
if (get(block, 'userId ') === this.state.selectedUser.id) 
```